pythonforandroid/util.py

```diff
@@ -62,79 +60,6 @@ def ensure_dir(filename):
         makedirs(filename)
 
 
-class JsonStore(object):
```

Comment:
I didn't find any entry in our code where we use this class, but maybe was introduced for some future feature...@inclement, is that the case? Can we safely remove this class `JsonStore` and the method `which`?

Reply:
I think this is inherited from the original fork of kivy-ios, no objection to deleting it.

Reply:
Nice cleanup :clap: 
I love tests for that :smile: